exchanges/graphcache/src/default-storage/index.ts

```diff
@@ -0,0 +1,142 @@
+import { SerializedEntries, SerializedRequest } from '../types';
+
+const DB_NAME = 'graphcache-v3';
+const ENTRIES_STORE_NAME = 'entries';
+const METADATA_STORE_NAME = 'metadata';
+
+const database$ = new Promise<IDBDatabase>((resolve, reject) => {
+  const req = indexedDB.open(DB_NAME, 1);
+
+  req.onerror = () => {
+    reject(req.error);
+  };
+
+  req.onsuccess = () => {
+    resolve(req.result);
+  };
+
+  req.onupgradeneeded = () => {
+    req.result.createObjectStore(ENTRIES_STORE_NAME);
+    req.result.createObjectStore(METADATA_STORE_NAME);
+  };
+});
+
+export const readMetadata = (): Promise<SerializedRequest[]> =>
+  database$
+    .then(
+      (database): Promise<SerializedRequest[]> => {
+        const transaction = database.transaction(
+          METADATA_STORE_NAME,
+          'readonly'
+        );
+        const store = transaction.objectStore(METADATA_STORE_NAME);
+        const request = store.get('metadata');
```

Comment:
```suggestion
        const request = store.get(METADATA_STORE_NAME);
```

Reply:
Oh I didn’t realise I could just reuse it! Good catch 😂